tree/ntuple/v7/inc/ROOT/RNTupleWriter.hxx

```diff
@@ -110,6 +111,18 @@ public:
    /// Fill an entry into this ntuple, but don't commit the cluster. The calling code must pass an RNTupleFillStatus
    /// and check RNTupleFillStatus::ShouldFlushCluster.
    void FillNoFlush(ROOT::REntry &entry, RNTupleFillStatus &status) { fFillContext.FillNoFlush(entry, status); }
+
+   /// Fill an RRawPtrWriteEntry into this ntuple.  This method will perform
+   /// a light check whether the entry comes from the ntuple's own model
```

Comment:
What exactly does "light check" mean? Is there something that's not checked that the caller should take of themselves?

Reply:
I think it just means that the check itself is not computationally expensive. The caller should not have to do specific checks in the code (unless they are weirdly mixing up entries from different models).

Reply:
I amended the documentation to talk about the model ID, and mention that the method throws an exception otherwise.